configure

```diff
@@ -1732,6 +1738,7 @@ putvar CFG_AARCH64_LINUX_ANDROID_NDK
 putvar CFG_ARM_LINUX_ANDROIDEABI_NDK
 putvar CFG_I686_LINUX_ANDROID_NDK
 putvar CFG_MANDIR
+putvar CFG_USING_LIBCPP
```

Comment:
They should be needed in `src/etc/mklldeps.py`...

Huh. Somehow, that part of mklldeps didn't make it. I'll get that fixed.